### PR TITLE
Docs: Add Token multisig examples

### PR DESCRIPTION
**Pull request by t-nelson**
#### Problem

Docs contain no examples demonstrating SPL Token multisig usage

#### Changes

Add some

### CODE REVIEW

File: docs/src/token.md [hunk 1]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
...
+Minting 1 tokens
+  Token: 4VNVRJetwapjwYU8jf4qPgaCeD76wyz8DuNj8yMCQ62o
+  Recipient: EX8zyi2ZQUuoYtXd4MKmyHYLTjqFdWeuoTHcsTdJcKHC
+Signature: 2ubqWqZb3ooDuc8FLaBkqZwzguhtMgQpgMAHhKsWcUzjy61qtJ7cZ1bfmYktKUfnbMYWTC1S8zdKgU6m4THsgspT
+```
+
+### Example: Mint with multisig authority and offline signers
+
+This example builds off of the [online mint with multisig](#example-mint-with-multisig-authority)
+example. Be sure to familiarize yourself with it, [offline signing](https://docs.solana.com/offline-signing)
```
Comment (CriesofCarrots) [inline docs/src/token.md:388]:
```suggestion
example. Be sure to familiarize yourself with that process, [offline signing](https://docs.solana.com/offline-signing),
```

File: docs/src/token.md [hunk 2]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
...
+- Token account owner authority: `spl-token transfer ...`, `spl-token approve ...`,
+`spl-token revoke ...`, `spl-token burn ...`, `spl-token wrap ...`,
+`spl-token unwrap ...`, `spl-token authorize ... owner ...`
+- Token account close authority: `spl-token close ...`, `spl-token authorize ... close ...`
+
+### Example: Mint with multisig authority
+
+First create keypairs to act as the multisig signer set. In reality. these can
+be any supported signer; Ledger hardware wallet, keypair file, paper wallet. For
+convenience, keypair files will be used in this example.
```
Comment (CriesofCarrots) [inline docs/src/token.md:306]:
```suggestion
First create keypairs to act as the multisig signer set. In reality, these can
be any supported signer, like: a Ledger hardware wallet, a keypair file, or a paper wallet. For
convenience, keypair files will be used in this example.
```

File: docs/src/token.md [hunk 3]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
...
+```
+
+### Example: Mint with multisig authority and offline signers
+
+This example builds off of the [online mint with multisig](#example-mint-with-multisig-authority)
+example. Be sure to familiarize yourself with it, [offline signing](https://docs.solana.com/offline-signing)
+and the [durable nonce](https://docs.solana.com/offline-signing/durable-nonce)
+feature before continuing.
+
+This example will use the same mint, token and multisig accounts as the online
```
Comment (CriesofCarrots) [inline docs/src/token.md:392]:
```suggestion
This example will use the same mint, token, and multisig accounts as the online
```

File: docs/src/token.md [hunk 4]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
...
+example. As well as multisig account signer set keypair file names.
```
Comment (CriesofCarrots) [inline docs/src/token.md:393]: "As well as multisig account signer set keypair file names." is a fragment, but I don't grok it well enough to suggest an edit I'm confident in. Should it be combined with the next sentence?
Something like: "In addition to the multisig signer keypair files, a nonce account at..."

File: docs/src/token.md [hunk 5]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
...
+$ solana nonce-account Fjyud2VXixk2vCs4DkBpfpsq48d81rbEzh6deKt7WvPj
+Balance: 0.01 SOL
+Minimum Balance Required: 0.00144768 SOL
+Nonce: 6DPt2TfFBG7sR4Hqu16fbMXPj8ddHKkbU4Y3EEEWrC2E
+Fee: 5000 lamports per signature
+Authority: 5hbZyJ3KRuFvdy5QBxvE9KwK17hzkAUkQHZTxPbiWffE
+```
+
+For the fee-payer and nonce-authority roles a local hot wallet at
+`5hbZyJ3KRuFvdy5QBxvE9KwK17hzkAUkQHZTxPbiWffE` is used.
```
Comment (CriesofCarrots) [inline docs/src/token.md:406]:
```suggestion
For the fee-payer and nonce-authority roles, a local hot wallet at
`5hbZyJ3KRuFvdy5QBxvE9KwK17hzkAUkQHZTxPbiWffE` will be used.
```

File: docs/src/token.md [hunk 6]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
...
+
+This example will use the same mint, token and multisig accounts as the online
+example. As well as multisig account signer set keypair file names.
+
+A nonce account at `Fjyud2VXixk2vCs4DkBpfpsq48d81rbEzh6deKt7WvPj` will be used
+```
+$ solana nonce-account Fjyud2VXixk2vCs4DkBpfpsq48d81rbEzh6deKt7WvPj
+Balance: 0.01 SOL
+Minimum Balance Required: 0.00144768 SOL
+Nonce: 6DPt2TfFBG7sR4Hqu16fbMXPj8ddHKkbU4Y3EEEWrC2E
```
Comment (CriesofCarrots) [inline docs/src/token.md:400]: You just updated this to "Nonce blockhash", didn't you?

File: docs/src/token.md [hunk 7]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
...
+
+For the fee-payer and nonce-authority roles a local hot wallet at
+`5hbZyJ3KRuFvdy5QBxvE9KwK17hzkAUkQHZTxPbiWffE` is used.
+
+First a template command is built by specifying all signers by their public
+key. Upon running this command, all signers will be listed as "Absent Signers"
+in the output. This command will be run by each offline signer to generate the
+corresponding signature.
+
+NOTE: The argument to the `--blockhash` parameter is the "Nonce:" field from
```
Comment (CriesofCarrots) [inline docs/src/token.md:413]: Nonce blockhash?

**Review by CriesofCarrots — COMMENTED** [review]
Examples look great! Just a few nits, plus one spot I didn't understand.

File: docs/src/token.md (hunk 4 above)
Comment (t-nelson) [inline docs/src/token.md:393]:
```suggestion
example, as well as the multisig account signer set keypair file names.
```

File: docs/src/token.md (hunk 6 above)
Comment (t-nelson) [reply docs/src/token.md:400]:
```suggestion
Nonce blockhash: 6DPt2TfFBG7sR4Hqu16fbMXPj8ddHKkbU4Y3EEEWrC2E
```

Good call!

File: docs/src/token.md (hunk 7 above)
Comment (t-nelson) [reply docs/src/token.md:413]:
```suggestion
NOTE: The argument to the `--blockhash` parameter is the "Nonce blockhash:" field from
```

File: docs/src/token.md (hunk 4 above)
Comment (t-nelson) [inline docs/src/token.md:393]:
```suggestion
This example will use the same mint account, token account, multisig account,
and multisig signer set keypair file names as the online example.
```

I think this fixes https://github.com/solana-labs/solana-program-library/pull/814/files#r520248724 (and adds the Oxford Comma from the previous suggestion)

File: docs/src/token.md (hunk 2 above)
Comment (t-nelson) [reply docs/src/token.md:306]:
```suggestion
First create keypairs to act as the multisig signer set. In reality, these can
be any supported signer, like: a Ledger hardware wallet, a keypair file, or
a paper wallet. For convenience, keypair files will be used in this example.
```

Same as yours, but wrapped at 80 char

Comment (CriesofCarrots) [reply docs/src/token.md]: 😳 sorry, laziness on the wrapping

File: docs/src/token.md (hunk 4 above)
Comment (CriesofCarrots) [reply docs/src/token.md:393]: A suggestion on your suggestion: `multisig signer-set keypair filenames`

Comment (CriesofCarrots) [reply docs/src/token.md:393]:
```suggestion
This example will use the same mint account, token account, multisig account,
and multisig signer-set keypair filenames as the online example.
```

Comment (t-nelson) [reply docs/src/token.md:393]: Sg!  I'll change the other "signer set" instances to match

File: docs/src/token.md [hunk 8]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
 1
 ```
 
+### Mutlisig usage
+
+The main difference in `spl-token` command line usage when referencing multisig
+accounts is in specifying the `--owner` argument. Typically the signer specified
+by this argument directly provides a signature granting its authority, but in
+the multisig case it just points to the address of the multisig account.
+Signatures are then provided by the multisig signer set members specified by the
```
Comment (t-nelson) [inline docs/src/token.md:289]:
```suggestion
Signatures are then provided by the multisig signer-set members specified by the
```

File: docs/src/token.md [hunk 9]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
...
+- Mint account freeze authority: `spl-token freeze ...`, `spl-token thaw ...`,
+`spl-token authorize ... freeze ...`
+- Token account owner authority: `spl-token transfer ...`, `spl-token approve ...`,
+`spl-token revoke ...`, `spl-token burn ...`, `spl-token wrap ...`,
+`spl-token unwrap ...`, `spl-token authorize ... owner ...`
+- Token account close authority: `spl-token close ...`, `spl-token authorize ... close ...`
+
+### Example: Mint with multisig authority
+
+First create keypairs to act as the multisig signer set. In reality, these can
```
Comment (t-nelson) [inline docs/src/token.md:304]:
```suggestion
First create keypairs to act as the multisig signer-set. In reality, these can
```

File: docs/src/token.md [hunk 10]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
...
+be any supported signer, like: a Ledger hardware wallet, a keypair file, or
+a paper wallet. For convenience, keypair files will be used in this example.
+```
+$ for i in $(seq 3); do solana-keygen new --no-passphrase -so "signer-${i}.json"; done
+Wrote new keypair to signer-1.json
+Wrote new keypair to signer-2.json
+Wrote new keypair to signer-3.json
+```
+
+In order to create the multisig account, the public keys of the signer set must
```
Comment (t-nelson) [inline docs/src/token.md:314]:
```suggestion
In order to create the multisig account, the public keys of the signer-set must
```

File: docs/src/token.md [hunk 11]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
...
+
+Now the multisig account can be created with the `spl-token create-multisig`
+subcommand. Its first positional argument is the minimum number of signers (`M`)
+that must sign a transaction affecting a token/mint account that is controlled
+by this multisig account. The remaining positional arguments are the public keys
+of all keypairs allowed (`N`) to sign for the multisig account. This example
+will use a "2 of 3" multisig account.  That is, two of the three allowed keypairs
+must sign all transactions.
+
+NOTE: SPL Token Multisig accounts are limited to a signing set of eleven signers
```
Comment (t-nelson) [inline docs/src/token.md:331]:
```suggestion
NOTE: SPL Token Multisig accounts are limited to a signer-set of eleven signers
```

File: docs/src/token.md [hunk 12]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
...
+```
+$ spl-token authorize 4VNVRJetwapjwYU8jf4qPgaCeD76wyz8DuNj8yMCQ62o mint 46ed77fd4WTN144q62BwjU2B3ogX3Xmmc8PT5Z3Xc2re
+Updating 4VNVRJetwapjwYU8jf4qPgaCeD76wyz8DuNj8yMCQ62o
+  Current mint authority: 5hbZyJ3KRuFvdy5QBxvE9KwK17hzkAUkQHZTxPbiWffE
+  New mint authority: 46ed77fd4WTN144q62BwjU2B3ogX3Xmmc8PT5Z3Xc2re
+Signature: yy7dJiTx1t7jvLPCRX5RQWxNRNtFwvARSfbMJG94QKEiNS4uZcp3GhhjnMgZ1CaWMWe4jVEMy9zQBoUhzomMaxC
+```
+
+To demonstrate that the mint account is now under control of the multisig
+account, attempting to mint with one (online) signer fails
```
Comment (t-nelson) [inline docs/src/token.md:362]:
```suggestion
account, attempting to mint with one multisig signer fails
```

File: docs/src/token.md [hunk 13]
```diff
@@ -280,6 +280,235 @@ $ spl-token supply 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
...
+$ spl-token mint 4VNVRJetwapjwYU8jf4qPgaCeD76wyz8DuNj8yMCQ62o 1 EX8zyi2ZQUuoYtXd4MKmyHYLTjqFdWeuoTHcsTdJcKHC \
+--owner 46ed77fd4WTN144q62BwjU2B3ogX3Xmmc8PT5Z3Xc2re \
+--multisig-signer signer-1.json
+Minting 1 tokens
+  Token: 4VNVRJetwapjwYU8jf4qPgaCeD76wyz8DuNj8yMCQ62o
+  Recipient: EX8zyi2ZQUuoYtXd4MKmyHYLTjqFdWeuoTHcsTdJcKHC
+RPC response error -32002: Transaction simulation failed: Error processing Instruction 0: missing required signature for instruction
+```
+
+But repeating with a second (again, online) signer, succeeds
```
Comment (t-nelson) [inline docs/src/token.md:373]:
```suggestion
But repeating with a second multisig signer, succeeds
```

**Comment by t-nelson** [comment]
Looks like Themis build's busted.  I'll take a look in the morning if no one beats me to it.

Travis CI docs build is green, so this is going in

**Pull request merged**